docs/7-dashboard-ui/3-Zarf-deployment-ui.md

```diff
@@ -0,0 +1,115 @@
+# Zarf Deployment Web UI
+
+Zarf has a Deoployment Web UI built in. The Deployment Web Ui supports a nubmer of Zarf features used during the package deployment process. For users who prefer not to use the command line tool, the Web UI creates a simple experience to deploy and manage Zarf clusters and packages. The Web UI can be used to connect to connect to existing clusters (via Kubeconfig), initalize a cluster, deploy packages into a cluster, update packages in the cluster, and remove packages from the cluster. 
+
+The Zarf Web UI mirros the functionality of the Zarf CLI commands, but with a more intuitive flow and familuar web application patterns for non-technical users. The web UI does not offer any additional commands or core functionaliity to Zarf. 
+
+## Open the Zarf Deployment Web UI
+
+The Zarf Deployment Web UI can be easily spun up with a single command from the CLI terminal. 
+
+Follow these steps to get started using the Web UI
+
+1. Step one: Install Zarf binary
```

Comment:
I think this should ideally be a link to the install page.

Reply:
@Racer159 Thank you for that catch! Agree.